alsa_settings/LNLM_SDW_AIOC.sh

```diff
@@ -4,6 +4,9 @@ set -e
 amixer -c sofsoundwire cset name='Headphone Switch' on
 amixer -c sofsoundwire cset name='rt711 FU05 Playback Volume' 80
 
+# override jack detection mode to headset
+amixer -c sofsoundwire ct name='rt711 GE49 Selected Mode' 2
```

Comment:
Maybe:
```suggestion
amixer -c sofsoundwire ct name='rt711 GE49 Selected Mode' 2 || true
```

Reply:
Good point, it will fail without the PR.